.lintrunner.toml

```diff
@@ -288,6 +288,7 @@ exclude_patterns = [
     'torch/csrc/lazy/**/*',
     'torch/csrc/onnx/init.cpp',
     'torch/csrc/mps/**/*',
+    'torch/csrc/xpu/**',
```

Comment:
Why do you want to exclude it from lintrunner?

Reply:
For this PR, there is no clangtidy error. So we remove this line from the exclude list.